sdk/servicebus/service-bus/README.md

```diff
@@ -62,6 +62,7 @@ You can instantiate this class using its constructors:
   - This method takes the host name of your Service Bus instance and a credentials object that you need
     to generate using the [@azure/identity](https://www.npmjs.com/package/@azure/identity)
     library. The host name is of the format `name-of-service-bus-instance.servicebus.windows.net`.
+    If you're using an own token provider against AAD, then set the "scopes" for service-bus to be `["https://servicebus.azure.net//user_impersonation"]` to get the appropriate token.
```

Comment:
```suggestion
    If you're using an own implementation of the `TokenCredential` interface against AAD, then set the "scopes" for service-bus to be `["https://servicebus.azure.net//user_impersonation"]` to get the appropriate token.
```

Reply:
@chradek Any suggestions for the above?

Reply:
`...using an own implementation of the TokenCredential interface from "@azure/identity", then set the...` ?

Reply:
No, the interface is not from `azure/identity`, only the implementations are from there.

> If you are using your own implementation of the `TokenCredential` interface against AAD, then set ..

Also, you need to do the same on the constructor overload for ServiceBusClient